go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/gballet/go-verkle
 
 go 1.18
 
-require github.com/crate-crypto/go-ipa v0.0.0-20230601170251-1830d0757c80
+require github.com/crate-crypto/go-ipa v0.0.0-20230626131944-6a9b06cf26df
```

Comment:
Reminder before merging: this is a tentative PR in go-ipa. Before pushing to `master`, I should open a PR there to merge it in go-ipa first.

Reply:
[Reference](https://github.com/gballet/go-verkle/pull/370)